data_steward/tools/snapshot_by_query.py

```diff
@@ -125,7 +147,14 @@ def create_snapshot_dataset(project_id, dataset_id, snapshot_dataset_id):
                         dest='snapshot_dataset_id',
                         help='Name of the new dataset that needs to be created',
                         required=True)
+    parser.add_argument('-ds',
+                        '--data_stage',
+                        action='store',
+                        dest='data_stage',
+                        help='Stage of the cdr generation',
+                        choices=['pre_deid', 'post_deid'],
+                        required=True)
```

Comment:
Instead of passing a parameter like this, can we make it check if the supplied input dataset has a person table with the extra columns and if it does, use the new person schema, else stick to the old one?

Reply:
I actually thought about getting from the table, but this is going to go away in the near future so I didn't spend much time to change it.

Reply:
It's a simpler change than what you are currently doing. All it needs is to query the dataset with `SELECT * from {proj}.{dataset}.INFORMATION_SCHEMA.COLUMNS` and look for `sex_at_birth_concept_id ` in the `result_df.['column_name']` column. You wouldn't have to modify any of the bash scripts and it is easy to miss that parameter while doing it manually.

Reply:
@nishanthpp93 Updated the data_stage issue